src/index.html

```diff
@@ -17,16 +17,38 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form data-qa="big">
+      <div class="search__big">
+        <img
+          src="images/Search.svg"
+          class="search__big-icon"
+          alt="search-vector"
+        />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+        <input
+          class="search__big-input"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
+    </form>
+
+    <form data-qa="small">
+      <div class="search__small">
+        <img
+          src="images/Search.svg"
+          class="search__small-icon"
+          alt="search-vector"
+        />
+
+        <input
+          class="search__small-input"
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
+    </form>
```

Comment:
Similar to the previous form, consider adding a `<label>` element for the input field to improve accessibility. This will make the form more accessible to users relying on screen readers.